addons/website_event/static/src/snippets/options.js

```diff
@@ -18,11 +18,12 @@ options.registry.WebsiteEvent = options.Class.extend({
             method: 'read',
             args: [
                 [this.eventId],
-                ['website_menu'],
+                ['website_menu', 'website_url'],
             ],
         });
-        this.data.reload = this.currentWebsiteUrl;
+        this.eventWebsiteUrl = rpcData[0]['website_url'];
```

Comment:
Don't need to set it as a "this" variable then, you can directly assign the value to data.reload.

Reply:
@awa-odoo Done GTG :)